### ISSUE TITLE
PTN000191653  GO:0004022 | alcohol dehydrogenase (NAD+) activity

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5494]
* **PTHR ID & PTN node:**

GO:0004022 | alcohol dehydrogenase (NAD+) activity | IBA with P08319 , P07327 , P00325 , P25437 , P40394 , PTN000191653 , MGI:1349472
91653) , [MGI:1349472](http://www.informatics.jax.org/accession/MGI:1349472)

* **Sequences with problematic annotation (ID + gene/protein name):**

https://www.pombase.org/gene/SPBC1539.07c

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

is physiologically  a glutathione-dependent formaldehyde dehydrogenase

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by ValWood** [comment]
ignore, bi functional
